src/app/components/SearchIcon.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Create one file called icons.tsx, put any svg code there, and export all of them as objects : {}